extensions/oauth/src/test/resources/authorization-code/authorization-code-localcallbackref-config.xml

```diff
@@ -14,22 +14,21 @@
 
     <oauth:token-manager-config name="tokenManagerConfig"/>
 
+    <oauth:authorization-code-grant-type
```

Comment:
I find it very weird that one annotation is the only change in the productive code. Seems to me like you're fixing the bug by simply defining the grant type as a top level element. If this is so, then we need to:

* Have @pablolagreca to validate that is fine
* Define the grant type not only as supports top level definition, you must also set it as doesn't support inline definition

If this is not the case, then you're missing tests in which you use it inline.